packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -148,11 +202,13 @@ abstract class FlutterCommand extends Command<Null> {
     if (commandPath != null)
       flutterUsage.sendCommand(usagePath);
 
-    await runCommand();
+    return await runCommand();
```

Comment:
I've heard that while the VM unwraps futures in these cases, it's bad style, but then I've also heard it's fiiiine.  @abarth or @Hixie, is there a definitive answer?